src/transformers/image_processing_utils_fast.py

```diff
@@ -279,8 +280,34 @@ def resize(
                 "Size must contain 'height' and 'width' keys, or 'max_height' and 'max_width', or 'shortest_edge' key. Got"
                 f" {size}."
             )
+        # This is a workaround to avoid a bug in torch.compile when dealing with uint8 on AMD MI3XX GPUs
+        # Tracked in PyTorch issue: https://github.com/pytorch/pytorch/issues/155209
+        # TODO: remove this once the bug is fixed (detected with torch==2.7.0+git1fee196, torchvision==0.22.0+9eb57cd)
+        if torch.compiler.is_compiling() and is_rocm_platform():
+            return self.compile_friendly_resize(image, new_size, interpolation, antialias)
```

Comment:
could you share what tests would need this?

Why we have compiling in image processing stage ..?

Reply:
Added the failing tests to the PR description! As for why there is compiling, I am not sure -- probably helps speedup the processing stage